webpack.config.js

```diff
@@ -47,6 +47,13 @@ module.exports = (env = {NODE_ENV: process.env.NODE_ENV || 'production'}) => ({
         export: 'default',
       },
     },
+    webexcc: {
```

Comment:
```suggestion
    cc: {
```

Reply:
or `contact-center: {`

Reply:
We cannot have a name with - in webpack. Gives error. And keeping it cc will become very generic hence decided to keep it as webexcc

Reply:
I'd like someone else to weigh in on this too. I feel like webex is redundany. Does `'contact-center'` as the key not work? It should work in JSON this way.

Reply:
<img width="728" alt="Screenshot 2024-10-10 at 10 22 21 PM" src="https://github.com/user-attachments/assets/bbe17cda-8ebf-4462-8c19-8c640a1f150c">

Reply:
We can keep this as any name for now. Let's change it once we get confirmation from Arun.